integration-tests/gradle/pom.xml

```diff
@@ -361,6 +361,16 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>jdk18-workarounds</id>
+            <activation>
+                <jdk>[18,)</jdk>
+            </activation>
+            <properties>
+                <!-- auto-exclude tests that are known to fail on JDK 18 or above (tagged via @Tag("failsOnJDK18")) -->
+                <excludedGroups>failsOnJDK18</excludedGroups>
+            </properties>
+        </profile>
```

Comment:
Last time I had this in `build-parent` (first for Java 16 then 17), but these two tests are the only ones that have to be excluded with that tag and so I added it "down" here.